lua/astrocommunity/completion/avante-nvim/init.lua

```diff
@@ -0,0 +1,97 @@
+return {
+  "yetone/avante.nvim",
+  build = ":AvanteBuild",
```

Comment:
```suggestion
  "yetone/avante.nvim",
  event = "VeryLazy",
  lazy = false,
  build = ":AvanteBuild",
```

Added in event and lazy = false from the docs.

Reply:
sry for the delay. thank you for your review. i'm not sure which way the project prefer since i was asked to remove these two lines and add commands before. so i'll leave it to you maintainers first. have a nice day

Reply:
Does the loading work in the newest spec? @vonhyou . From my own experience, it might not when you're using plugins written in multiple languages.

Reply:
Hi, when I tested in my environment, the plugin did load after entering the cmds according to lazy profile. But I never tried windows or other platforms. I think it's reasonable to go with your way